Src/Campus.Master.API/UI/src/app/registration-page/registration-page.component.html

```diff
@@ -16,33 +16,41 @@ <h1 class="registration__title">Getting started with <span>Campus</span></h1>
                  [ngClass]="{ invalid: registerForm.get('first_name').touched && registerForm.get('first_name').invalid }">
               <label for="first-name" class="first-name"></label>
               <input id="first-name" type="text" placeholder="First Name" formControlName="first_name">
-              <div
-                *ngIf="registerForm.get('first_name').touched && registerForm.get('first_name').invalid"
+              <div *ngIf="registerForm.get('first_name').touched && registerForm.get('first_name').invalid"
                 class="validation">
-                <small *ngIf="registerForm.get('first_name').errors.required">
+                <div [ngSwitch]="registerForm.get('first_name').errors | firstError">
+                  <small *ngSwitchCase="'required'">
                   First name is required
                 </small>
-                <small *ngIf="registerForm.get('first_name').errors.pattern">
-                  Only english letters
+                  <small *ngSwitchCase="'maxlength'">
+                    First name > 50 symbols
                 </small>
+                  <small *ngSwitchCase="'pattern'">
+                    Only English letters
```

Comment:
It's better to use message **Only Latin characters are allowed**

Reply:
Replaced with more short message, due to small gap between messages in Fist/LastName fields

Reply:
If so, you can replace the message with smth like this one **Only Latin**